scripts/globals/job_utils/monk.lua

```diff
@@ -51,9 +52,20 @@ xi.job_utils.monk.useChakra = function(player, target, ability)
         end
     end
 
+    local monkLevel = 0
+
+    if target:getMainJob() == xi.job.MNK then
+        monkLevel = target:getMainLvl()
+    else
+        monkLevel = target:getSubLvl()
+    end
```

Comment:
We are using this logic a lot, I think we also use it in other places.
Maybe we should just turn it to an util in the globals/utils.lua
Like...

```lua
function utils.getCurrentLevel(actor, job)
    local level = 0
    
    if actor:getMainJob() == job then
        level = actor:getMainLvl()
    elseif actor:getSubJob() == job then
        level = actor:getSubLvl()
    end

    return level
end
```

Probably better for a different PR, but putting it there

Reply:
Done

Reply:
Oh, while I was looking at my diff I noticed that function already exists as `utils.GetActiveJobLevel`